api/institutions/views.py

```diff
@@ -420,3 +421,43 @@ def get_default_queryset(self):
     # overrides RetrieveAPIView
     def get_queryset(self):
         return self.get_queryset_from_request()
+
+class InstitutionUserMetricsList(JSONAPIBaseView, ListFilterMixin, generics.ListAPIView, InstitutionMixin):
+
+    permission_classes = {
+        drf_permissions.IsAuthenticatedOrReadOnly,
+        base_permissions.TokenHasScope,
+        IsInstitutionalMetricsUser,
+    }
+
+    required_read_scopes = [CoreScopes.INSTITUTION_METRICS_READ]
+    required_write_scopes = [CoreScopes.NULL]
+
+    serializer_class = InstitutionUserMetricsSerializer
+
+    view_category = 'institutions'
+    view_name = 'institution-user-metrics'
+
+    def get_default_queryset(self):
+        institution = self.get_institution()
+        queryset = MockQueryset()
+
+        results = UserInstitutionProjectCounts.get_current_user_metrics(institution)
+        for user_record in results:
+            record_dict = {}
+            user_id = user_record.user_id
+            fullname = OSFUser.objects.get(guids___id=user_id).fullname
+            record_dict['user'] = f'({user_id}) {fullname}'
```

Comment:
Should this just be `fullname` here? Or is this what the requirements/mockups called for?

Reply:
This was based on the requirements document "List of names of all SSO users with user GUID in parentheses, first and last names (full name) linked to their OSF public profile" https://docs.google.com/document/d/15CqCveNHy9vu62vGWv_tuLk9K6mzHDuxcW5A-EvNS0I